sympy/integrals/__init__.py

```diff
@@ -9,7 +9,11 @@
     >>> integrate(sin(x),x)
     -cos(x)
 """
+from .deltafunctions import deltaintegrate
+from .heurisch import heurisch, heurisch_wrapper
 from .integrals import integrate, Integral, line_integrate
+from .manualintegrate import manualintegrate, integral_steps
+from .rationaltools import ratint
```

Comment:
Am 09.03.2015 um 15:00 schrieb Sergey B Kirpichev:
>>   from .integrals import integrate, Integral, line_integrate
>> +from .manualintegrate import manualintegrate, integral_steps
>> +from .rationaltools import ratint
>
>> What do you mean with '"shortened" import'?
>
> e.g. "from sympy.integrals import deltaintegrate" instead of what was quoted above.

I'm assuming you want ".integrals" changed to "sympy.integrals".
I see no reason to do that, given that the pre-existing imports in that 
file use relative imports, too.

(If my assumption is wrong and you want the list of names shortened, 
please tell what you'd like to have seen, and what you did see instead.)